### ISSUE TITLE
Too few arguments to function matomo_reports_user_presave()

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/matomo_reports#1]
Testing this module and when I try to create a user while it's installed I get a fatal error:

> Too few arguments to function matomo_reports_user_presave(), 1 passed in /app/core/includes/module.inc on line 960 and exactly 3 expected

**Comment by oadaeh** [comment]
Thanks for creating this issue.
I have committed a fix to the dev branch.
Please test it and let me know if it works for you, and if it does, I'll create a 1.x-4.0.1 tag.
Please also note that I did commit some changes to the menu structure, so you might need a cache clear for those changes to take effect: https://github.com/backdrop-contrib/matomo_reports/commit/0ffabb48a77ab0ae1c18de2b47e7043b56af3c80

**Comment by gitressa** [comment]
I also saw this bug, and can confirm that the https://github.com/backdrop-contrib/matomo_reports/commit/0ffabb48a77ab0ae1c18de2b47e7043b56af3c80 commit 16 Jan 2021 fixes the issue, and I can create a user. Thanks!

PS. I can't see a dev-branch when I clone the repo, so maybe the update got committed into the default `1.x-4.x` branch?:
```
$ git clone https://github.com/backdrop-contrib/matomo_reports.git 
$ cd matomo_reports
$ git branch -a
* 1.x-4.x
  remotes/origin/1.x-4.x
  remotes/origin/HEAD -> origin/1.x-4.x
```

**Comment by oadaeh** [comment]
There is only the 1.x-4.x branch. That is currently what is used for dev.
I just created a [1.x-4.0.1 release](https://github.com/backdrop-contrib/matomo_reports/releases/tag/1.x-4.0.1) that includes the fix for this, so I'm closing this issue.
Edit: It is also already available on the Backdrop CMS site: https://backdropcms.org/project/matomo_reports

**Comment by gitressa** [comment]
Thanks for releasing the update so quickly @oadaeh. I tried uninstalling, removing and reinstalling the module, but still got `4.0.0` I also tried updating via `/admin/config/system/updates`. Even though `4.0.1` is listed, it is still ends up at `4.0.0` after update completion ...

Name | Project type | Project status | Installed version | Recommended version
-- | -- | -- | -- | --
Matomo Reports | Module | Enabled | 1.x-4.0.0 | 1.x-4.0.1 (release notes)

It's probably some caching somewhere, I'll give it a try later.
